packages/create-gatsby/src/cmses.json

```
@@ -0,0 +1,7 @@
+{
+  "gatsby-source-wordpress-experimental": { "message": "WordPress" },
+  "gatsby-source-contentful": { "message": "Contentful" },
+  "gatsby-source-sanity": { "message": "Sanity" },
+  "gatsby-source-shopify": { "message": "Shopify" },
+  "gatsby-source-prismic-graphql": { "message": "Prismic GraphQL" }
```

```suggestion
  "gatsby-source-prismic": { "message": "Prismic" }
```

The official plugin is https://github.com/angeloashmore/gatsby-source-prismic

https://prismic.io/docs/the-future-of-prismic-and-gatsby

---

This is the plugin that appeared in looker data, the beta one. If we're not comfortable using that, that's fine. It's likely that a number of people haven't migrated/found the latest, but it's something to note.

---

In my opinion, the Prismic docs don't instill a lot of confidence in either plugin: https://prismic.io/docs/technologies/the-future-of-prismic-and-gatsby (other link above is broken).

`gatsby-source-prismic-graphql` is being deprecated too.

Maybe we just don't include Prismic at all if we want to make the sure the first time experience of auto-setup is a good one?

---

I'm inclined to agree with @gillkyle here. Let's remove Prismic